dependencies.props

```diff
@@ -1,14 +1,14 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
   <PropertyGroup>
-    <CoreFxCurrentRef>2718aebf9298724ee788f871287bdc82ec950722</CoreFxCurrentRef>
+    <CoreFxCurrentRef>6de42fd82cb7b8db12fe9339e4304f77343edc83</CoreFxCurrentRef>
     <CoreClrCurrentRef>2718aebf9298724ee788f871287bdc82ec950722</CoreClrCurrentRef>
     <WCFCurrentRef>43f8d65aadc9dd4b30cb976652961bbbdcd15f5f</WCFCurrentRef>
   </PropertyGroup>
 
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
-    <CoreFxExpectedPrerelease>beta-24911-02</CoreFxExpectedPrerelease>
+    <CoreFxExpectedPrerelease>preview2-25226-01</CoreFxExpectedPrerelease>
```

Comment:
I really want us to get preview1 build first so please take dependency on corefx preview1 build for now. Once we get good WCF preview1 build we will move to preview2 shortly.